buildmaster/old_new_porting_map.yml

```diff
@@ -0,0 +1,382 @@
+# Mapping between old and new commondata for oldcommondata_porter
+#
+# <old_commondata_name>:
+#   new_name: <new_name>
+#   reference_arxiv: <http://etc>
+#   reference_hepdata: <http://etc>
+#
+
+# HERA
+HERACOMBCCEM:
+  new_name: HERA_CC_318GEV_EM-SIGMARED
+  reference_arxiv: '1506.06042'
+  reference_hepdata: 10.17182/hepdata.68951.v1/t7
+  journal: Eur. Phys. J. C75 (2015) 580
+HERACOMBCCEP:
+  new_name: HERA_CC_318GEV_EP-SIGMARED
+  reference_arxiv: '1506.06042'
+  reference_hepdata: 10.17182/hepdata.68951.v1/t6
+  journal: Eur. Phys. J. C75 (2015) 580
+HERACOMBNCEM:
+  new_name: HERA_NC_318GEV_EM-SIGMARED
+  reference_arxiv: '1506.06042'
+  reference_hepdata: 10.17182/hepdata.68951.v1/t5
+  journal: Eur. Phys. J. C75 (2015) 580
+HERACOMBNCEP460:
+  new_name: HERA_NC_225GEV_EP-SIGMARED
+  reference_arxiv: '1506.06042'
+  reference_hepdata: 10.17182/hepdata.68951.v1/t4
+  journal: Eur. Phys. J. C75 (2015) 580
+HERACOMBNCEP575:
+  new_name: HERA_NC_251GEV_EP-SIGMARED
+  reference_arxiv: '1506.06042'
+  reference_hepdata: 10.17182/hepdata.68951.v1/t3
+  journal: Eur. Phys. J. C75 (2015) 580
+HERACOMBNCEP820:
+  new_name: HERA_NC_300GEV_EP-SIGMARED
+  reference_arxiv: '1506.06042'
+  reference_hepdata: 10.17182/hepdata.68951.v1/t2
+  journal: Eur. Phys. J. C75 (2015) 580
+HERACOMBNCEP920:
+  new_name: HERA_NC_318GEV_EP-SIGMARED
+  reference_arxiv: '1506.06042'
+  reference_hepdata: 10.17182/hepdata.68951.v1/t1
+  journal: Eur. Phys. J. C75 (2015) 580
+HERACOMB_SIGMARED_B:
+  new_name: HERA_NC_318GEV_EAVG-SIGMARED-BOTTOM
+  reference_arxiv: '1804.01019'
+  reference_hepdata: ''
+  journal: Eur. Phys. J. C78 (2018) 473
+HERACOMB_SIGMARED_C:
+  new_name: HERA_NC_318GEV_EAVG-SIGMARED-CHARM
+  reference_arxiv: '1804.01019'
+  reference_hepdata: ''
+  journal: Eur. Phys. J. C78 (2018) 473
+HERAF2CHARM:
+  new_name: HERA_NC_318GEV_F2_EAVG-SIGMARED
+  set_name: HERA_NC_318GEV_F2
+  reference_arxiv: '1804.01019'
+  reference_hepdata: ''
```

Comment:
I don't recognize this dataset - what is it? The paper is the usual heavy pair production one (see entries above) and I think that has only the reduced cross sections for ccbar and bbar (also I just had a quick glance to the paper - i.e. we use table 2+3); moreover, if there would be a F2c measurement - that would go *instead* of `SIGMARED` (and not in parallel); as an aside: why this one does not have the journal?

Reply:
Some datasets are added just because it existed before. These are those for which either there is a `metadata.txt` file or were in the file that @enocera sent me.

>why this one does not have the journal?

Because I didn't have the information.

Reply:
This is something we should talk about... do we want to drop old data?

(if so that would make my life much easier...)

(I tried to do a lot of this automatically, there's a lot of chances for arxiv numbers and other stuff to be wrong!)

Reply:
My recommendation would be to forget about all pre-NNPDF4.0 data.

Reply:
I mean, all pre-NNPDF4.0 data sets that were not included in NNPDF4.0.

Reply:
Yes, of course. For now I've been checking things with theory 717. I'll also go back to theory 200 to make sure that I haven't missed anything that might have changed name on the meantime.